projects/plugins/backup/jetpack-backup.php

```diff
@@ -35,15 +35,9 @@
 
 // Constant definitions.
 define( 'JETPACK_BACKUP_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
-define( 'JETPACK_BACKUP_PLUGIN_ROOT_FILE', __FILE__ );
 define( 'JETPACK_BACKUP_PLUGIN_ROOT_FILE_RELATIVE_PATH', plugin_basename( __FILE__ ) );
-define( 'JETPACK_BACKUP_PLUGIN_SLUG', 'jetpack-backup' );
-define( 'JETPACK_BACKUP_PLUGIN_NAME', 'Jetpack Backup' );
-define( 'JETPACK_BACKUP_PLUGIN_URI', 'https://jetpack.com/jetpack-backup' );
 define( 'JETPACK_BACKUP_REQUIRED_JETPACK_VERSION', '10.0' );
 define( 'JETPACK_BACKUP_PLUGIN_FOLDER', dirname( plugin_basename( __FILE__ ) ) );
-define( 'JETPACK_BACKUP_PROMOTED_PRODUCT', 'jetpack_backup_t1_yearly' );
-define( 'JETPACK_BACKUP_DB_VERSION', '2' );
 
 /**
  * Checks if Jetpack is installed and if yes, require version 10+
```

Comment:
Do we really need this check here? And if we do, shouldn't we hook it into `init` hook and only run it if `is_admin()`?

Reply:
Also, there might be a less complicated way to check if the plugin is properly built?

Reply:
When Backup was originally constructed, it was very broken when installed alongside versions of Jetpack before 10.0 (what was released just prior to Backup's first release). Instead of figuring out backcompat with older versions, we just declared that Jetpack 10+ was required for Backup.

The built checks were mimicked off of what's in `jetpack.php` for Jetpack. Not saying there is or isn't an easier way to do it. In reality, the only time it would fail is when developing and forgetting to build the plugin so it's meant to help the dev experience.

Reply:
I think there's room for improvement there, but wanted to touch it in a separate PR